internal/cmd/login.go

```diff
@@ -84,12 +63,40 @@ $ infra login`,
 				options.Server = args[0]
 			}
 
+			if options.Server == "" {
+				return Error{
+					Message: "No server specified. Usage: infra login SERVER",
+				}
+			}
```

Comment:
If we're planning on doing something like "lookup the last server from their local config when not set" in a follow up, then I think this diff makes sense.

If we're not planning that, maybe we should change `Args: MaxArgs(1)` above to instead be `Args: ExactArgs(1)` to avoid this custom handling?